$NetBSD: patch-configure,v 1.1 2023/08/14 07:38:37 nros Exp $

* Fix compatibility problems
* IRIX: accept irix* hosts (czmq_prelude.h detects IRIX itself).

--- configure.orig
+++ configure
@@ -13094,7 +13094,7 @@
 fi
 
 
-if test "x${CZMQ_GCOV}" == "xyes"; then
+if test "x${CZMQ_GCOV}" = "xyes"; then
     CFLAGS="-O0 -g -fprofile-arcs -ftest-coverage"
 
     if test "x${CZMQ_ORIG_CFLAGS}" != "xnone"; then
@@ -13132,7 +13132,7 @@
 fi
 
 
-if test "x${CZMQ_ASAN}" == "xyes"; then
+if test "x${CZMQ_ASAN}" = "xyes"; then
     CFLAGS="${CFLAGS} -fsanitize=address"
     CXXFLAGS="${CXXFLAGS} -fsanitize=address"
 
@@ -13171,7 +13171,7 @@
 fi
 
 
-if test "x$ZPROJECT_BINDINGS_PYTHON" == "xyes"; then
+if test "x$ZPROJECT_BINDINGS_PYTHON" = "xyes"; then
      if true; then
   ENABLE_BINDINGS_PYTHON_TRUE=
   ENABLE_BINDINGS_PYTHON_FALSE='#'
@@ -15589,6 +15589,8 @@
 $as_echo "#define CZMQ_HAVE_AIX 1" >>confdefs.h
 
         ;;
+    *irix*)
+        ;;
     *hpux*)
         # Define on HP-UX to enable all library features
         CPPFLAGS="-D_POSIX_C_SOURCE=200112L"
